pkgs/applications/misc/tqsl/cmake_lib_path.patch

```diff
@@ -0,0 +1,12 @@
+diff -dur tqsl-2.3.1/src/CMakeLists.txt tqsl-2.3.1b/src/CMakeLists.txt
+--- tqsl-2.3.1/src/CMakeLists.txt	2017-04-17 20:53:22.000000000 -0400
++++ tqsl-2.3.1b/src/CMakeLists.txt	2017-10-05 21:14:39.048329343 -0400
+@@ -54,7 +54,7 @@
+ if(NOT APPLE AND NOT WIN32)
+ set_source_files_properties(location.cpp PROPERTIES COMPILE_DEFINITIONS CONFDIR="${CMAKE_INSTALL_PREFIX}/share/TrustedQSL/")
+ set(HEADERS_TO_INSTALL tqsllib.h tqslerrno.h cabrillo.h adif.h tqslconvert.h)
+-install(TARGETS tqsllib DESTINATION lib$(LIB_SUFFIX))
++install(TARGETS tqsllib DESTINATION lib${LIB_SUFFIX})
```

Comment:
This might be also worth reporting upstream.

Reply:
Already have. They don't seem to have an open bug tracker, but there's a help email and a trouble ticket form. I submitted to both.

Reply:
Just dug this up: https://sourceforge.net/p/trustedqsl/mailman/message/35820793/

They've already fixed it. Just haven't released the changes yet.